mcnp/test/CMakeLists.txt

```diff
@@ -5,17 +5,24 @@ set(LINK_LIBS)
 list(APPEND LINK_LIBS gtest)
 list(APPEND LINK_LIBS pthread)
 list(APPEND LINK_LIBS dagmciface)
+list(APPEND LINK_LIBS dagtally)
 list(APPEND LINK_LIBS ${DAG_LINK_LIBS})
 
 # Include directories
 include_directories(..)
+include_directories(../../common)
 include_directories(${MOAB_INCLUDE_DIRS})
 include_directories(${HDF5_INCLUDE_DIRS})
 include_directories(${GTEST_INCLUDE_DIR})
 
 # Setup unit tests
+set(TEST_NAME dagmcnp_unit_tests)
 set(DRIVER mcnp5_unit_test_driver.cc)
-setup_test(dagmcnp_unit_tests cpp ${DRIVER} "${LINK_LIBS}")
```

Comment:
To elaborate, mcnp_funcs.cpp needs to be in there when you don't build DAG-MCNP for some reason